app/javascript/lib/validation/validate-file-content.js

```diff
@@ -387,14 +405,27 @@ export async function parseFile(file, fileType, fileOptions={}, sizeProps={}) {
 
         parseResult.issues.push(['warn', 'incomplete:range-request', msg])
       }
+      let chunker
+      if (!fileInfo.isGzipped) {
+        chunker = new ChunkedLineReader(file, ignoreLastLine)
+      } else {
+        chunker = new ChunkedLineReader(file, ignoreLastLine, true)
+      }
 
-      const chunker = new ChunkedLineReader(file, ignoreLastLine)
       const { issues, delimiter, numColumns } =
         await parseFunctions[fileType](chunker, fileInfo.fileMimeType, fileOptions)
       fileInfo.linesRead = chunker.linesRead
       fileInfo.delimiter = delimiter
       fileInfo.numColumns = numColumns
       parseResult.issues = parseResult.issues.concat(issues)
+
+      if (fileInfo.isGzipped && fileInfo.linesRead === GZIP_MAX_LINES + 1) {
+        const msg =
+        'Due to this file\'s size, it will be fully validated after upload, ' +
+        'and any errors will be emailed to you.'
```

Comment:
I think backticks are preferred over '+' for multi-line strings

Reply:
I vaguely recall trailing spaces in multi-line strings require a special solution with backticks, making them less an obvious choice here.